jekyll/_cci2/server/v4.4/overview/circleci-server-overview.adoc

```diff
@@ -83,13 +89,18 @@ The application exposes a number of external ports. These ports are used for var
 | Only required for the Nomad clients.
 |===
 
-CircleCI server schedules CI jobs using the link:https://www.nomadproject.io/[Nomad] scheduler. The Nomad control plane runs inside of Kubernetes, while the Nomad clients, which are responsible for running scheduled CircleCI jobs, are provisioned outside the cluster. CircleCI
-server can run Docker jobs on the Nomad clients themselves or in a dedicated virtual machine (VM).
+[#job-scheduling-and-artifact-storage]
+=== Job scheduling and artifact storage
+
+CircleCI server schedules CI jobs using HashiCorp link:https://www.nomadproject.io/[Nomad]. The Nomad control plane runs inside your Kubernetes cluster, while the Nomad clients, which are responsible for running scheduled CircleCI jobs, are provisioned outside the cluster.
```

Comment:
This is less true now. We do still use Nomad, but not like we used to. In the past the VM jobs were orchestrated through Nomad but that is no longer the case.
Now we have three different systems that jobs can utilize:
* Nomad
* VMs (via Machine Provisioner)
* Runner

Reply:
@ryanwohara so Nomad is just for Docker executor?